From a2a74022e2db0cc44ab68a846c3acafd9df1d971 Mon Sep 17 00:00:00 2001
From: Will Woods <wwoods@redhat.com>
Date: Tue, 24 Jan 2012 13:09:34 -0500
Subject: [PATCH] netroot: actually run netroot hooks

The line "source_all netroot" was trying to source netroot hooks from
/netroot, which doesn't exist, so netroot hooks were never executed.

Signed-off-by: Will Woods <wwoods@redhat.com>
---
 modules.d/40network/netroot |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/40network/netroot b/modules.d/40network/netroot
index 9d996cc..bda34df 100755
--- a/modules.d/40network/netroot
+++ b/modules.d/40network/netroot
@@ -123,7 +123,7 @@ if [ -n "$dest" ] && ! arping -q -f -w 60 -I $netif $dest ; then
 fi
 
 # Source netroot hooks before we start the handler
-source_all netroot
+source_all $hookdir/netroot
 
 # Run the handler; don't store the root, it may change from device to device
 # XXX other variables to export?
